gunrock/app/sm/sm_app.cu

```diff
@@ -245,10 +242,36 @@ double sm(
     double elapsed_time = gunrock_sm(parameters, data_graph, query_graph, subgraphs);
     // Cleanup
     data_graph.Release();
+    query_graph.Release();
 
     return elapsed_time;
 }
 
+
+/*
+ * @brief Simple interface take in graph as CSR format
+ * @param[in]  num_nodes   Number of veritces in the input graph
+ * @param[in]  num_edges   Number of edges in the input graph
+ * @param[in]  row_offsets CSR-formatted graph input row offsets
+ * @param[in]  col_indices CSR-formatted graph input column indices
+ * @param[in]  edge_values CSR-formatted graph input edge weights
+ * @param[in]  num_runs    Number of runs to perform SM
+ * @param[out] subgraphs   Return number of subgraphs
+ * \return     double      Return accumulated elapsed times for all runs
+ */
+double sm(
+    const int            num_nodes,
+    const int            num_edges,
+    const int           *row_offsets,
+    const int           *col_indices,
+    const unsigned long *edge_values,
+    const int            num_runs,
+          int           *subgraphs)
+{
+    return sm(num_nodes, num_edges, row_offsets, col_indices,
```

Comment:
why have `num_runs` as an argument if it's hardcoded to 1?

Reply:
I think here is only a default value. @neoblizz correct me if I'm wrong. Other apps do so as well.